bin/nanocl/src/models/stats.rs

```diff
@@ -0,0 +1,7 @@
+use clap::Parser;
+
+#[derive(Default, Parser)]
+pub struct StatsArg {
```

Comment:
`StatsOpts` we use the `Arg` naming when it's a command that have subcommand

Reply:
Changed the name.